iphone/cli/hooks/package.js

```diff
@@ -131,10 +132,11 @@ exports.init = function (logger, config, cli) {
 
 					// move the finished archive directory into the correct location
 					fs.ensureDirSync(archivesDir);
-					appc.fs.copyDirSyncRecursive(stagingArchiveDir, dest, {
-						logger: logger.debug
-					});
-
+					try  {
+						fs.move(stagingArchiveDir, dest);
+					} catch (error){
+						logger.error(__('Failed to to move archive to correct location'));
+				    }
```

Comment:
<!--
  1 failure:  iphone/cli/hooks/...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: iphone/cli/hooks/package.js;
  Line: 139;
-->
- :no_entry_sign: iphone/cli/hooks/package.js line 139 – Mixed spaces and tabs. (no-mixed-spaces-and-tabs)